### DISCUSSION TITLE
How do I setup device to subscribe to grouptopic2-4?

### DISCUSSION THREAD

**Discussion by mvadu** [arendst/Tasmota#20270] [Q&A] [answered]
I see that in #8014  GroupTopic1-4 were added, but kept off by default. I don't see any way to turn them on. I have added `GroupTopic2 admin` to all my devices which are in various groups (e.g. ` tasmota/globe-light/globe-light2/stat/RESULT = {"GroupTopic1":"globe-light","GroupTopic2":"admin","GroupTopic3":"","GroupTopic4":""}`) so that I can send any admin commands like OTA update via this topic. But I don't see any of the devices responding to any commands for that admin topic. They do respond for GroupTopic1.

**Comment by sfromis** [comment, marked as answer]
As you have deviated from the standard topic structure (like setting a FullTopic), you need to make sure that your way of doing that works. With `WebLog 3` activated, at boot time you can see what topic masks Tasmota subscribes to.

**Reply by mvadu** [reply]
you are right.. it is indeed subscribing to the group topic2, but given I have a fixed part in the full topic, the group2 comes after the fixed part which is unique to this group. So I can't think of a way to have one group across all tasmota devices, and one to group similar devices (usually from same manufacturer). But your suggestion showed what is going on.. 

```
21:30:14.107 MQT: Subscribe to tasmota/globe-light/globe-light2/cmnd/#
21:30:14.109 MQT: Subscribe to tasmota/globe-light/globe-light/cmnd/#
21:30:14.112 MQT: Subscribe to tasmota/globe-light/admin/cmnd/#
21:30:14.115 MQT: Subscribe to cmnd/globe-light2_fb/#
```

**Reply by sfromis** [reply]
If you want a topic structure allowing the group topic to be across the 2nd level, you could try if it works for you with topics like `globe-light/globe-light2` including your 2nd level, instead of pushing this into the full topic mask, when you do not want this to actually be the "full topic".
